.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "\U0001F5CEDocumentation"
```

Comment:
When I see this icon in my MacBook, it looks like a simple square rather than a document.
What do you think of the icon of an open book [`\U0001f4d6`](https://www.fileformat.info/info/unicode/char/1F4D6/index.htm) or memo [`\U0001f4dd`](https://www.fileformat.info/info/unicode/char/1F4DD/index.htm) as an alternative?
They seem to be used in GitHub emoji API as [book](https://github.githubassets.com/images/icons/emoji/unicode/1f4d6.png?v8) and [pencil](https://github.githubassets.com/images/icons/emoji/unicode/1f4dd.png?v8), respectively.


The current icon in MacBook (High Sierra)
![image](https://user-images.githubusercontent.com/3255979/67823743-94b8e980-fb07-11e9-8f03-90c404bf37fc.png)

Open book
![image](https://user-images.githubusercontent.com/3255979/67824644-57a22680-fb0a-11e9-9217-bbfeee5f461d.png)

Memo
![image](https://user-images.githubusercontent.com/3255979/67824618-41946600-fb0a-11e9-9492-3e8e7d43b297.png)

Reply:
Thank you for the important feedback!
I replaced the current icon with [open book](https://github.githubassets.com/images/icons/emoji/unicode/1f4d6.png?v8) (https://github.com/pfnet/optuna/pull/652/commits/5d30b7862f434c13c3e9e0032aa60df51c5c3a85).